.github/workflows/udeps.yml

```diff
@@ -28,17 +28,19 @@ jobs:
         toolchain: nightly
         override: true
 
-    - name: Cache cargo registry
+    - name: Cache CARGO_HOME
       uses: actions/cache@v3
       with:
-        path: ~/.cargo/registry
-        key: ubuntu-latest-nightly-cargo-registry-${{ hashFiles('**/Cargo.toml') }}
-
-    - name: Cache cargo index
-      uses: actions/cache@v3
-      with:
-        path: ~/.cargo/git
-        key: ubuntu-latest-nightly-cargo-index-${{ hashFiles('**/Cargo.toml') }}
+        path: |
+          ~/.cargo/bin/
+          ~/.cargo/registry/index/
+          ~/.cargo/registry/cache/
+          ~/.cargo/git/db/
+        # Add date to the cache to keep it up to date
```

Comment:
What's this comment about ? I don't see any date ?

Reply:
Ooof good catch. The way we're doing this actually isn't the best, because we're compiling `udeps` with the nightly toolchain (so caching might be useless because the toolchain may change every night and cause `udeps` to be rebuilt). It only needs to be _run_ with the nightly toolchain but it can be built with stable. I'll re-work this a bit